app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] The class name for users controller should be `UsersController`. Please make sure to change the class name to `UsersController`.